
PLAY [Ubuntu-20-CP] ************************************************************

TASK [Change /tmp permission] **************************************************
[0;31mAn exception occurred during task execution. To see the full traceback, use -vvv. The error was: PermissionError: [Errno 1] Operation not permitted: b'/tmp'[0m
[0;31mfatal: [Ubuntu-20-CP]: FAILED! => {"changed": false, "module_stderr": "Shared connection to 192.168.56.35 closed.\r\n", "module_stdout": "Traceback (most recent call last):\r\n  File \"/home/devops/.ansible/tmp/ansible-tmp-1646620219.914263-24535-146504839545642/AnsiballZ_file.py\", line 107, in <module>\r\n    _ansiballz_main()\r\n  File \"/home/devops/.ansible/tmp/ansible-tmp-1646620219.914263-24535-146504839545642/AnsiballZ_file.py\", line 99, in _ansiballz_main\r\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\r\n  File \"/home/devops/.ansible/tmp/ansible-tmp-1646620219.914263-24535-146504839545642/AnsiballZ_file.py\", line 47, in invoke_module\r\n    runpy.run_module(mod_name='ansible.modules.file', init_globals=dict(_module_fqn='ansible.modules.file', _modlib_path=modlib_path),\r\n  File \"/usr/lib/python3.8/runpy.py\", line 207, in run_module\r\n    return _run_module_code(code, init_globals, run_name, mod_spec)\r\n  File \"/usr/lib/python3.8/runpy.py\", line 97, in _run_module_code\r\n    _run_code(code, mod_globals, init_globals,\r\n  File \"/usr/lib/python3.8/runpy.py\", line 87, in _run_code\r\n    exec(code, run_globals)\r\n  File \"/var/tmp/ansible_ansible.builtin.file_payload_we4n1i0l/ansible_ansible.builtin.file_payload.zip/ansible/modules/file.py\", line 972, in <module>\r\n  File \"/var/tmp/ansible_ansible.builtin.file_payload_we4n1i0l/ansible_ansible.builtin.file_payload.zip/ansible/modules/file.py\", line 958, in main\r\n  File \"/var/tmp/ansible_ansible.builtin.file_payload_we4n1i0l/ansible_ansible.builtin.file_payload.zip/ansible/modules/file.py\", line 675, in ensure_directory\r\n  File \"/var/tmp/ansible_ansible.builtin.file_payload_we4n1i0l/ansible_ansible.builtin.file_payload.zip/ansible/module_utils/basic.py\", line 1178, in set_fs_attributes_if_different\r\n  File \"/var/tmp/ansible_ansible.builtin.file_payload_we4n1i0l/ansible_ansible.builtin.file_payload.zip/ansible/module_utils/basic.py\", line 938, in set_mode_if_different\r\nPermissionError: [Errno 1] Operation not permitted: b'/tmp'\r\n", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}[0m

PLAY RECAP *********************************************************************
[0;31mUbuntu-20-CP[0m               : ok=0    changed=0    unreachable=0    [0;31mfailed=1   [0m skipped=0    rescued=0    ignored=0   

